.gitignore

```diff
@@ -16,3 +16,4 @@ charts/**/*.lock
 
 PRIVATEKEY
 PUBLICKEY
+examples/tmp
```

Comment:
Wondering why ignore `examples/tmp`?

Reply:
Just some temporary test examples for yourself, stored locally, but do not wish to commit

Reply:
Recommend: the `.gitignore` just store some common files.

Reply:
yes, removed, thanks